static/src/stylesheets/_vars.scss

```diff
@@ -114,6 +114,12 @@ $frontend-palette: (
     crosswordAccent1:       guss-colour(multimedia-support-1, $pasteup-palette),
     crosswordAccent2:       guss-colour(news-main-2, $pasteup-palette),
 
+    sudoku-mute:            lighten(guss-colour(news-main-2, $pasteup-palette), 30%),
+    sudoku-accent:          lighten(guss-colour(news-main-2, $pasteup-palette), 10%),
+
+    button-mute:            guss-colour(neutral-7, $pasteup-palette),
+    button-accent:          #d7d7d7,
```

Comment:
Sorry bro, no actual hex values in here. Either use a colour function or add it to pasteup-palette. Although I'm still confused as to where these colours have come from.

How are button colours specified else where? Do we need new vars for them?